internal/command/deploy/machines_deploymachinesapp.go

```diff
@@ -311,8 +311,45 @@ type metadata struct {
 	value string
 }
 
+func (md *machineDeployment) guestForMachine(mach *api.Machine, group string) *api.MachineGuest {
```

Comment:
Hey, I don't think this is a good idea. If I read this correctly, it will update existing machines but the original intention of `--vm-*` flags was to serve as backup for `fly scale vm` command when it is not possible to set vm size before first deploy.

Reply:
yeap, I think we have to keep its original meaning and add a better text description to the flags.